test/Microsoft.AspNet.Mvc.TagHelpers.Test/AnchorTagHelperTest.cs

```diff
@@ -73,7 +76,7 @@ public class AnchorTagHelperTest
             Assert.Equal("myanchor", attribute.Value);
             attribute = Assert.Single(output.Attributes, kvp => kvp.Key.Equals("href"));
             Assert.Equal("home/index", attribute.Value);
-            Assert.Equal("Something", output.Content);
+            Assert.Equal("Something", output.Content.ToString());
```

Comment:
avoid `ToString()` and use `Get()` instead in all tests.  override `ToString()` to improve the situation when debugging but don't call it in our code (except perhaps in one or two `TagHelperOutput` tests that confirm our debuggability hasn't regressed).